src/components/Content/terminologies.json

```
@@ -1,23 +1,110 @@
 {
-  "ahkaam": "rulings",
-  "fard": "obligatory",
-  "ghusl": "full ablution",
-  "hajj": "pilgrimage",
-  "hayd": "menstruation",
-  "istihadah": "dysfunctional bleeding",
-  "kabah": "the sacred structure located in the center of the Masjid al-Haram in Mecca, Saudi Arabia",
-  "ma’dhur": "excused person",
-  "makrooh": "disliked",
-  "mustahabb": "recommended",
-  "nifaas": "postnatal bleeding",
-  "qadaa": "makeup",
-  "salaah": "prayer",
-  "shari’iyyah": "Islamic law",
-  "sunnah": "the teachings, practices, and examples set forth by the Prophet SAW",
-  "takbeer-e-tahreema": "the opening phrase of the prayer",
-  "tawaaf": "circumambulation around the Kabah",
-  "tuhr": "purity",
-  "umrah": "minor pilgrimage",
-  "wajib": "necessary",
-  "wudu": "partial ablution"
+  "ahkaam": {
+    "clarification": "rulings",
```

let's future proof translations:
```suggestion
    "clarification": {
       "en": "rulings",
     }
```